scalalib/src/mill/scalalib/dependency/metadata/MetadataLoader.scala

```diff
@@ -0,0 +1,7 @@
+package mill.scalalib.dependency.metadata
```

Comment:
`MetadataLoader.scala`, `MavenMetadataLoader.scala` and `MetadataLoaderFactory.scala` exist in sbt-updates, be I rewrote evrything in thoses classes, so I didn't copy the BSD license text in those files